sktime/forecasting/model_selection/_tune.py

```diff
@@ -267,7 +267,10 @@ def evaluate_candidates(candidate_params):
         # Select n best forecaster
         self.n_best_forecasters_ = []
         self.n_best_scores_ = []
-        for i in range(self.return_n_best_forecasters):
+        n = self.return_n_best_forecasters
```

Comment:
can we think about a better name than `n`?

Reply:
idk like `_forecasters_to_return`

Reply:
I can change it to `local_return_n_best_forecasters = self.return_n_best_forecasters`?